src/utils/callbacks.js

```diff
@@ -0,0 +1,24 @@
+const FacebookStrategy = require('passport-facebook').Strategy;
+const User = require('../service/Users/users');
+
+module.exports = {
+	facebook: (config) => new FacebookStrategy(
+		config,
+		async (token, refreshToken, profile, done) => {
```

Comment:
we should use refreshtoken also as access token are short lived token and as per facebook refresh token are long lived token and these token are refreshed every day by the auth servers

Reply:
Yeah , I am parking all those changes for future work  / next week.